examples/Elastic.Xunit.ExampleComplex/TestWithoutClusterFixture.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace Elastic.Xunit.ExampleComplex
 {
-	[IntegrationTestCluster(typeof(TestCluster))]
```

Comment:
Reviewing note: this removes support for marking clusters through attributes. In the spirit of simplifying marking test classes with `IClusterFixture<>` is now the sole way to get clusters injected. 

This was introduced in: https://github.com/elastic/elasticsearch-net/pull/3424

Since this was not ported to the new tests for the new client I think its safe to remove.